src/controllers/user.js

```diff
@@ -80,3 +80,15 @@ export const updateById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to update user')
   }
 }
+
+export const updateUserCohortById = async (req, res) => {
+  const { id, cohort_id: cohortId } = req.body
+
+  if (!cohortId) {
+    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+  }
+
+  const updateUser = await User.updateUsersCohort(Number(id), Number(cohortId))
```

Comment:
It would be worth adding a try/catch around the database call here to prevent any prisma errors causing unhandled exceptions.